docs/source/tutorials/GEDI/GEDI_L4A_Example.ipynb

```diff
@@ -0,0 +1,583 @@
+{
```

Comment:
<p>Just as a nice way for users to see more directly how the output file corresponds to the <code>columns</code> and <code>query</code> specified as inputs, consider adding the following as the last line in the <code>else</code> block: <code>gedi_gdf.head()</code>.     </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I've put it in next cell as <code>matplotlib</code> will overwrite the output.  </p>

Reply:
<p>Unfortunately, that fails if the <code>else</code> block is not executed since <code>gedi_gdf</code> is not defined in that case.</p><p> </p><p>Instead, remove the additional cell, and instead, change line 11 in the preceding cell to </p><pre class="ql-syntax" spellcheck="false">print(gedi_gdf.head())
</pre><p>This will allow the data to be printed <em>and</em> have the plot show up below it. </p><p> </p>

Reply:
Oh I am so sorry, I missed this latest conversation.